wildfly-users-jpa/src/main/java/model/BaseEntity.java

```diff
@@ -30,12 +30,10 @@ public void setUuid(String uuid) {
 	
 	@PrePersist
 	public void onPrePersist() {
-		ensureUuid();
+		getUuid();

```

Comment:
forditva -> az ensureUuid() metodusban a null check es a getUuid()-ben csak felhivod az ensureUuid()-t:
`	public String getUuid() {
		ensureUuid();
		return this.uuid;
	}
	
	@PrePersist
	public void onPrePersist() {
		getUuid();
	}
	
	private void ensureUuid() {
			setUuid (UUID.randomUUID().toString());
	}`